### ISSUE TITLE
Add confirmation when user save to log

### ISSUE THREAD

**Issue by lineaba** [prog-o-meter/prog-o-meter#91]
We have a new logging features, which lets the user save text to a log. When the user clicks save, there is no visual confirmation that the text have been saved, which might make the user unsure whether there changes have been saved. 
Can we add some sort of confirmation, either a popup, or a little checkmark or something, that lets the user know their changes have been added?
Alternatively, we could also just have the window close on save.
Anybody have any suggestions for how to handle this? @sdkwok you added the feature, have you had any thoughts regarding this?

**Comment by sdkwok** [comment]
Hi @lineaba, just for simplicity's sake I didn't add such a feature at the time, although it did cross my mind. I can take a look at this again and try adding a popup to alert the user. Although it does seem like a message or a checkmark in the text window might be less obtrusive.

**Comment by lineaba** [comment]
Yes, I think a small checkmark or animation, or save on close (maybe button
text can just be save&close) would be preferable to a pop-up.

**Comment by omkar2810** [comment]
Can I work on this. 
Actually I have also commented on another issue(restart meter) ,so just let me know which one is available to work.
PS - I would be great if you could explicitly mention if someone is working on a given issue , I am finding it a little bit difficult to know whether I should work on some issue or not as someone else might be working on it already.
Also let me know if there is already a way for this and I am missing it,
 Thanks.

**Comment by lineaba** [comment]
Hey @omkar2810, we gotta ask whether @sdkwok got started on this, if not, you can go ahead. I will give an update as soon as possible.


Regarding whether someone is working on a specific issue, my best suggestion is reading through the comments in the issue For example you can see above that @sdkwok says they'll work on it. I can't always guarantee that someone will finish something, just because they say that they will start it, but I try to go through the issues and get an update on them when there haven't been activity in a while. I will comment if the issue was not yet solved, even though someone say they would solve it, which means someone new can get started.
Github also has an assigning function which we try to use, but sometimes it doesn't work, because the user have not yet accepted my invitation to be a contributor, or I haven't had a moment to send the contributor-invitation out.
I know it is frustrating that things aren't 100% clear, but it is something I think follows when a group of strangers start working together on a project from all around the world. I try to do what I can to keep things clear, but it doesn't always work. My best advice is to ask if you are in doubt. Also know that you are always welcome to work on any issue at any time, but you are just not guaranteed that your code will be merged in (added to the codebase), since someone might have already added a solution, that is currently being reworked to be ready to merge.
This was a very long answer - maybe I will take parts of it and add it to the contribution guide to makes this clearer to new contributors. Please let me know if this was helpful, or if you have any more questions :)

**Comment by omkar2810** [comment]
Thanks @lineaba ,I completely understand your point. Just wanted to know the correct way of approaching the issues . 
Anyways, I am completely fine with what you have suggested and will try to work accordingly.

**Comment by lineaba** [comment]
Hey @omkar2810 I just spoke to @sdkwok. You can go ahead and start working on the issue. I suggest you go the route of doing a save&close button, but if you can find a way to graphically give information that something have been closed (with a checkmark or something like that) that is cool too. We just wanna avoid another pop-up window. Let me know if you have any questions :)

**Comment by omkar2810** [comment]
Ok @lineaba I take this issue.
Just to be sure , I have to integrate the save and close buttons into a single "save and close" button and on pressing the button there should be some sort of indication(preferably an animation or a check mark on the main window itself) that the log has been saved successfully,

**Comment by lineaba** [comment]
Combining save and Close will be sufficient. The animation was an
alternative to save and close, where each would be their own button, and
when you save you would see an animation to know save had been successful.
But I personally prefer the save and close option (without animation).
